stubs/inertia/resources/js/Pages/Teams/TeamMemberManager.vue

```diff
@@ -59,7 +59,7 @@
                 </template>
 
                 <template #actions>
-                    <jet-action-message :on="addTeamMemberForm.recentlySuccessful" class="mr-3">
+                    <jet-action-message :on="addTeamMemberFormRecentlySuccessful" class="mr-3">
```

Comment:
Another possible naming change.
```suggestion
                    <jet-action-message :on="successfullyAddedTeamMember" class="mr-3">
```